physics/lsm_ruc.F90

```diff
@@ -365,18 +367,20 @@ subroutine lsm_ruc_run                                            & ! inputs
       implicit none
 
 !  ---  constant parameters:
-      real(kind=kind_phys), parameter :: rhoh2o  = 1000.0
-      real(kind=kind_phys), parameter :: stbolt  = 5.670400e-8
+      real(kind_phys), parameter :: rhoh2o  = 1000.0
+      real(kind_phys), parameter :: stbolt  = 5.670400e-8
```

Comment:
This should be passed in from the host:
`[con_sbc]`
 `standard_name = stefan_boltzmann_constant`
 `long_name = Stefan-Boltzmann constant`
 `units = W m-2 K-4`
 `dimensions = ()`
 `type = real`
 `kind = kind_phys`

Reply:
Changed to pass it from the host.

Reply:
Use con_sbc for Stefan-Boltzman constant.